app/components/UI/TooltipButton.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box, Text } from 'rebass'
+
+const StyledTooltipIconBox = styled(Box)`
+  border-radius: 50%;
+  height: 16px;
+  line-height: 16px;
+  margin-left: 5px;
+  position: relative;
+  text-align: center;
+  transform: translateZ(0);
+  width: 16px;
+  z-index: 2;
+
+  &:hover {
+    cursor: pointer;
+  }
+`
+StyledTooltipIconBox.displayName = 'TooltipIconBox'
+
+const StyledTooltipWrapper = styled(Box)`
+  border: 1px solid white;
+  border-radius: 4px;
+  padding: 5px 5px 5px 25px;
+  position: absolute;
+  top: -5px;
+  width: 225px;
+  z-index: 1;
+  transform: translateZ(0);
+`
+StyledTooltipWrapper.displayName = 'TooltipWrapper'
+
+/**
+ * @render react
+ * @name TooltipButton
+ * @example
+ * <TooltipButton>Some explanation text here</TooltipButton>
+ */
+class TooltipButton extends React.Component {
```

Comment:
Let's call this `Tooltip` rather than `TooltipButton` since it encompasses both the button and the tooltip.